system/lib/libc/musl/src/internal/atomic.h

```diff
@@ -194,7 +194,7 @@ static inline void a_store(volatile int *p, int v)
 
 #ifndef a_barrier
 #define a_barrier a_barrier
-static void a_barrier()
+static inline void a_barrier()
```

Comment:
Is this going to be a pain when we update musl?

Reply:
No, musl patches are maintained in git these days and we have scripts for pushing and pulling them,